CHARTER.md

```diff
@@ -0,0 +1,71 @@
+# Aggie Robotics Charter
+
+## Preamble
+
+**Whereas** the students of the College of Engineering at North Carolina Agricultural and Technical State University seek opportunities to innovate and create in pursuit of understanding,
+
+**Whereas** opportunities for hands on application of engineering are not readily available to underclassmen,
+
+**Whereas** the Department of Electrical and Computer Engineering seeks to encourage students to apply their knowledge and seek new technical skills,
+
+**We**, the undersigned, found on this day the Aggie Robotics Club.
+
+## Mission
+
+The mission of Aggie Robotics is to provide practical educational opportunities and resources, for free innovation in: Robotics, Electrical Engineering, and Computer Science.
+
+## Membership
+
+Membership in Aggie Robotics contains 3 levels: Apprentice Membership, Journeyman Membership, and Crackerjack Membership. All members will receive a `Member Badge`. Members can advance in level by `badging`. The `badging` process will be defined in the by-laws and published openly.
```

Comment:
Okay, let's discuss this idea. We basically have two options for how to name members:

Description | Version One | Version Two
------------|------------|-----------
Entry-level, created their badge | Journeyman Member | Apprentice Member
Full members, but not CoE | Associate Member | Journeyman Member
Full members in the CoE | Full Member | Crackerjack Member

I propose version 1, @NowYouCanFindMe proposes version 2, and I invite him to write a quick note describing his reasoning.

@uriahnewkirk @agwaddel @Kennyal3 what are your thoughts?